lnbits/wallets/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .corelightningrest import CoreLightningRestWallet
 from .eclair import EclairWallet
 from .fake import FakeWallet
+from .alby import AlbyWallet
```

Comment:
This list hast to be sorted, otherwise `ruff` complains.

Please move `Alby` import just before `Cliche` import.

Reply:
@prusnak this is still.... a draft PR i'm still working on it, should i make it officially active for review? As it seems like you're very enthusiatically reviewing it before i'm done testing.....?